rotkehlchen/globaldb/handler.py

```diff
@@ -1961,6 +1961,9 @@ def asset_id_exists(identifier: str, use_packaged_db: bool = False) -> str:
     @staticmethod
     def get_collection_main_asset(identifier: str) -> str | None:
         """
+        TODO: ADjust this. We need a better approach. When you do remove the exceptions
```

Comment:
nitpick for next PR: Fix typo and add this issue: https://github.com/rotki/rotki/issues/8639

(I know I wrote this)